add pre-setup instructions for minsta (#92)
Mintbase/minsta: README.md

This repository hosts the primary codebase for the [Minsta App](https://minsta.me), originally created and developed by [@microchipgnu](https://github.com/microchipgnu).

All new Minsta instances should be derived from this repository.

[Video Tutorial on Setting up](https://www.loom.com/share/d5a038fb341c40be9ae131dd82f199a80)

## Pre-Setup - Proxy Contract

> 1. You got to have a NEAR Contract where you add a proxy contract as a minter.

Reviewer: This sounds a bit too informal: suggest replacing "got" with "need"
```suggestion
1. You need to have a NEAR Contract where you add a proxy contract as a minter.
```

> 2. The proxy contract is meant for let any user who is not a minter, to be able to mint images on Minsta.

Reviewer: ```suggestion
2. The proxy contract is meant to allow any user who is not a minter, to be able to mint images on Minsta.
```

> 3. The Near Contract will be where the NFT images will be minted

Reviewer: I am not sure I understand this sentence... dos this mean "the near contract supplied in step 1 will be the mint address of the image"?

I thought images were stored off-chain.

4. The proxy contract will be the minter
5. The user wallet address will be the owner of the NFT.

### Deploying a Near Contract on Mintbase:
1. Login on Mintbase and access [Contracts Page](https://www.mintbase.xyz/launchpad/contracts/0)
2. Click on New Contract

> 3. Choose Store Name (this will be the contract address to add on your minsta instance, this need to be added on the `process.env.NEXT_PUBLIC_NFT_CONTRACT_ADDRESS` env var.) and Store Symbol

Reviewer: ```suggestion
3. Choose Store Name (this will be the contract address to add on your minsta instance, this need to be added on the `process.env.NEXT_PUBLIC_NFT_CONTRACT_ADDRESS` environment variable) and Store Symbol
```

4. Proceed to transaction.
5. Succeeded
6. Go to Contract Settings

### Add Proxy Minter Contract
1. Under Contract Settings go to Minters

> 2. add `0.drop.proxy.mintbase.near` ( this is the contracct address that need to be added on `process.env.NEXT_PUBLIC_PROXY_MINTER_CONTRACT_ADDRESS`), and click Add Minters

Reviewer: ```suggestion
2. add `0.drop.proxy.mintbase.near` (this is the contract address that need to be added on `process.env.NEXT_PUBLIC_PROXY_MINTER_CONTRACT_ADDRESS`), and click Add Minters.
```

3. Proceed to transaction.

> 54. Succeeded

Reviewer: ```suggestion
4. Succeeded
```
